src/languages/en.js

```diff
@@ -520,7 +520,7 @@ export default {
         passwordFormTitle: 'Welcome back to the New Expensify! Please set your password.',
         passwordNotSet: 'We were unable to set your new password. We have sent you a new password link to try again.',
         setPasswordLinkInvalid: 'This set password link is invalid or has expired. A new one is waiting for you in your email inbox!',
-        validatingAccount: 'Verifying account',
+        validateAccount: 'Verify account',
```

Comment:
This text is used in the button you press after you click the validation link sent to your email to set your password.